src/main/java/com/synopsys/integration/jira/common/rest/service/PluginManagerService.java

```diff
@@ -85,6 +85,21 @@ public PluginManagerService(Gson gson, JiraHttpClient httpClient, JiraService ji
         return Optional.empty();
     }
 
+    public boolean isAppInstalled(String username, String accessTokenOrPassword, String appKey) throws IntegrationException {
+        String apiUri = createBaseRequestUrl() + appKey + "-key";
+        Request.Builder requestBuilder = createBasicRequestBuilder(apiUri, username, accessTokenOrPassword);
+        requestBuilder.addQueryParameter(QUERY_KEY_OS_AUTH_TYPE, QUERY_VALUE_OS_AUTH_TYPE);
+        requestBuilder.method(HttpMethod.GET);
+        requestBuilder.addAdditionalHeader("Accept", MEDIA_TYPE_PLUGIN);
+
+        Response response = jiraService.get(requestBuilder.build());
+        if (response.isStatusCodeError()) {
```

Comment:
Just to be thorough, if the app is not installed, what error code do we get? I'm wondering if we should throw an exception for all error codes that aren't that one so that we don't relay the wrong information.

e.g. If the "not installed" error code would be a 404, then a 400, 401, 403, 503 could all still be cases that the plugin is installed, but there is something else going on.

Reply:
If we do decide to throw an exception, can we make it an `IntegrationRestException`?

Reply:
When the App is not installed we get a 404 error.